client/src/main/java/connection/Client.java

```diff
@@ -32,20 +32,25 @@
    */
   public void run() {
     try {
-      int port = setUpPort();
-      Connector connector = SocketConnector.from(host, port);
+      String[] configValues = setUpGame();
+      int port = Integer.parseInt(configValues[1]);
+      String IP = String.valueOf(configValues[0]);
+      Connector connector = SocketConnector.from(IP, port);
       in = MessageIn.from(connector);
       out = MessageOut.from(connector);
     } catch (IOException e) {
       LOGGER.log(Level.SEVERE, e.getMessage());
     }
   }
 
-  private int setUpPort() throws IOException {
+  private String[] setUpGame() throws IOException {
     Properties properties = new Properties();
     InputStream config = ClassLoader.getSystemResourceAsStream(SERVER_CONFIG_FILE);
     properties.load(config);
-    return Integer.parseInt(properties.getProperty("portNumber"));
+    String[] configValues = new String[2];
+    configValues[0] = properties.getProperty("IP");
+    configValues[1] = properties.getProperty("portNumber");
```

Comment:
We're using it in two methods so maybe create fields

Reply:
Or you could consider map instead of array, or just return properties object. Because now you getting two properties to just put it to array and relay on its order.